client/landing/gutenboarding/components/plans/plans-button/index.tsx

```diff
@@ -18,7 +18,7 @@ import { useSelectedPlan } from '../../../hooks/use-selected-plan';
 import { useCurrentStep, Step } from '../../../path';
 import { Plans } from '@automattic/data-stores';
 
-const PLANS = Plans.STORE_KEY;
+const PLANS = Plans.register();
```

Comment:
As a pattern in Gutenboarding we're doing this only in `stores/[DATA_STORE_NAME]` and import the store key from there where needed.